modules/1_opencv/src/opencv_coins.cpp

```diff
@@ -3,12 +3,28 @@
 using namespace cv;
 
 unsigned countCoins(const Mat& img) {
-    Mat gray, thresh;
```

Comment:
please change count spaces, because you created redundant differences

Reply:
i haven't changed this lines. Idnk why they are different

Reply:
maybe need replace spaces and tabs

Reply:
i tried, there are similar spaces, no tabs. In last commit i paste the part in the beginning, but there are no changes

Reply:
Try to find corresponding setting in Visual Studio (replace Tabs to Whitespace or something like that)

Reply:
done

Reply:
Please keep 4 whitepaces

Reply:
please use `git diff` that you local check differences

Reply:
Or just look at final version at https://github.com/dkurt/openvino_practice/pull/100/files

Reply:
fixed. i had 2 default spaces